### ISSUE TITLE
CenterNet Model Card

### ISSUE THREAD

**Issue by Sudharsan-V** [tenstorrent/tt-metal#20342]
Architecture: [link](https://docs.google.com/document/d/1guocK7-feh2WrH9_631odI0M_41ZfUWzY6pVze2Wh6s/edit?usp=sharing)

Source link - https://github.com/open-mmlab/mmdetection/tree/main

Operations:

- Conv2d
- BatchNorm2d
- ReLU

unit_test branch - Yet to Update

**Comment by saichandax** [comment]
@Sudharsan-V ,
Please add the link to the reference implementation branch.

**Comment by Sudharsan-V** [comment]
The reference(PyTorch) pipeline is completed.
Corresponding [commit](https://github.com/tenstorrent/tt-metal/compare/main...sudharsan/torch_centernet)

**Comment by saichandax** [comment]
@Sudharsan-V , please update the current status here. Thank you.

**Comment by saichandax** [comment]
@Sudharsan-V , please update the current status as executive summary in the description. Thank you.

**Comment by Sudharsan-V** [comment]
The pipeliene is updated with the demo and the draft PR is created [#20342](https://github.com/tenstorrent/tt-metal/pull/23397)
